File: gatling-performance-suite/src/test/scala/com/example/performance/scenarios/HomepageScenario.scala
package com.example.performance.scenarios

import io.gatling.core.Predef._
import io.gatling.http.Predef._

object HomepageScenario {

  val homepage = scenario("Homepage Scenario")
    .exec(
      http("Open Homepage")
        .get("/")
        .check(status.is(200))
    )
}

File: src/test/java/com/example/performance/scenarios/HomepageScenario.java
package com.example.performance.scenarios;

import io.gatling.javaapi.core.ChainBuilder;
import static io.gatling.javaapi.core.CoreDsl.exec;
import static io.gatling.javaapi.http.HttpDsl.http;
import static io.gatling.javaapi.http.HttpDsl.status;

public class HomepageScenario {

    public static ChainBuilder homepage = exec(
            http("Open Homepage")
                    .get("/")
                    .check(status().is(200))
    );
}